data-science-onramp/data-ingestion/setup_test.py

```diff
@@ -118,6 +125,10 @@ def test_setup():
     blob = get_blob_from_path(output_location)
     out = blob.download_as_string().decode("utf-8")
 
+    # check that tables were printed
+    for table_name in TABLE_NAMES:
+        assert table_printed(table_name, out)
```

Comment:
Is there another way to test that the table actually printed instead of checking that the print statement you added is present? There could be a possibility where the table actually did not print but your print statement did which is not a sufficient check.

Reply:
If the df.show prints column names, you could check for those